### DISCUSSION TITLE
Is there way to safe current session?

### DISCUSSION THREAD

**Discussion by mihailigoshin** [kovidgoyal/kitty#7890] [Q&A] [answered]
Hello, 

Is there a way to save opened tabs and executed ssh sessions? Like gnome-terminal profiles.

The use case is that I have a bunch of opened tabs and ssh sessions but I need to restart my workstation and I don't want to manually log to all servers again.

**Comment by kovidgoyal** [comment]
search the kitty documentation for the word session

**Reply by mihailigoshin** [reply]
I guess there is no dynamic way to save to setup.

**Comment by kovidgoyal** [comment, marked as answer]
Search google for kitty save session
https://github.com/dflock/kitty-save-session

**Reply by mihailigoshin** [reply]
Thank you.
